src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.CrossTargeting.targets

```diff
@@ -105,4 +105,11 @@ Copyright (c) .NET Foundation. All rights reserved.
   -->
   <Target Name="GetPackagingOutputs" />
 
+  <!-- This exists as a workaround for https://github.com/Microsoft/msbuild/issues/3558 -->
+  <PropertyGroup Condition="'$(DefaultProjectTypeGuid)' == ''">
+    <DefaultProjectTypeGuid Condition="'$(MSBuildProjectExtension)' == '.csproj'">{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</DefaultProjectTypeGuid>
+    <DefaultProjectTypeGuid Condition="'$(MSBuildProjectExtension)' == '.vbproj'">{F184B08F-C81C-45F6-A57F-5ABD9991F28F}</DefaultProjectTypeGuid>
+    <!-- Note: F# sets DefaultProjectTypeGuid in the F# SDK -->
```

Comment:
Wait, which F# SDK do you mean by this?  Didn't we fold things into the core SDK now?  @KevinRansom ?

Reply:
Do the templates actually reference that though?  I thought we kept that for the few people who took a dependency on it in the 1.1 timeframe, but we know just pattern F# after C# and VB.

Reply:
The templates do not.  The .NET Core SDK does internally reference it when it's an fsproj.

Reply:
Great - thanks for re-assuring me :)